src/cryptsetup/cryptsetup-generator.c

```diff
@@ -795,10 +795,56 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
         return 0;
 }
 
+static int add_crypttab_device(const char *name, const char *device,  const char *keyspec, const char *options) {
```

Comment:
BTW, we usually suffix the function name with `_one`. But feel free to ignore.

Reply:
Yeah, I know... It's just that `add_crypttab_devices_one` looked weird to me, so I "reduced" it. But if you prefer that other name, I can change it.